docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,274 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
+########################################
+
+With the Courses API **Courses** resource, you can complete the
+following tasks.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Get a list of courses:
+
+**********************
+Get a List of Courses
+**********************
+
+The endpoint to get a list of courses is ``/api/courses/v1/courses/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of the courses that are visible to a specific user. If the request
+is made by an anonymous user, a username is not required.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
+
+GET /api/courses/v1/courses/
+
+=====================
+Query Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+* org (optional) - A code for an organization; case-insensitive. Example:
+  "HarvardX". If ``org`` is specified, the list of courses only includes
+  courses belonging to the specified organization.
+
+* mobile (optional) - If specified, the list of courses only includes courses
+  that are designated as ``mobile_available``.
+
+
+=====================
+Response Values
+=====================
+
+The following fields are returned with a successful response.
+
+* blocks_url: Used to fetch the course blocks.
+
+* effort: A textual description of the weekly hours of effort expected in the
+  course.
+
+* end: The date on which the course ends.
+
+* enrollment_end: The date on which enrollment ends.
+
+* enrollment_start: The date on which enrollment begins.
+
+* id: A unique identifier of the course; a serialized representation of the
+  opaque key identifying the course.
+
+* media: An object that contains named media items, including the following
+  objects.
+
+  * course_image: An image to show for the course.
+
+    * uri: The location of the image.
+
+  * course_video: A video about the course.
+
+    * uri: The location of the video.
+
+  * image: URLs for images in the course, including ``raw``, ``small``, or
```

Comment:
@clintonb To clarify the question, the question isn't regarding the image types (raw, small, large).  My question is why we have 2 separate fields that have the same semantical name?  `course_image` and `image`?  Do we really need both?  If so, we should have different names for them since `course_image` is the same as `image` and would be confusing to users of the API.